ansible/roles/ocp-workload-aiadv-fuseonline/ilt_provision.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+END_PROJECT_NUM=1
+START_PROJECT_NUM=1
+WORKLOAD="ocp-workload-fuse-ignite"
```

Comment:
should it be ocp-workload-aiadv-fuseonline here ?

Reply:
@fridim Thanks, addressed using this commit 20f8459